[DependencyInjection] Prepending extension configs with file loaders (#52843)
symfony/symfony: src/Symfony/Component/DependencyInjection/CHANGELOG.md

---

 * Add argument `$prepend` to `ContainerConfigurator::extension()` to prepend the configuration instead of appending it
 * Add argument `$prepend` to `FileLoader::construct()` to prepend loaded configuration instead of appending it

>  * [BC BREAK] The `ContainerConfigurator::import()` method now prepend extension configs instead of appending it, but only when used in the `AbstractBundle::prependExtension()` and `AbstractExtension::prependExtension` methods

Reviewer: ```suggestion
 * [BC BREAK] When used in the `prependExtension()` methods, the `ContainerConfigurator::import()` method now prepends the configuration instead of appending it
 ```
